DuckDuckGo/Onboarding/ContextualOnboarding/ContextualOnboardingDialogs.swift

```diff
@@ -147,34 +147,52 @@ struct OnboardingFireButtonDialogContent: View {
     }()
 
     let viewModel: OnboardingFireButtonDialogViewModel
+    @State private var showNextScreen: Bool = false
 
     var body: some View {
-        ContextualDaxDialogContent(
-            orientation: .horizontalStack(alignment: .center),
-            message: attributedMessage,
-            messageFont: OnboardingDialogsContants.messageFont,
-            customActionView: AnyView(actionView))
+        VStack {
```

Comment:
out of curiosity, do we need the VStack here?

Can we just do 
```
if showNextScreen {
   OnboardingFinalDialogContent(highFiveAction: viewModel.highFive)
} else {
    ContextualDaxDialogContent(
         orientation: .horizontalStack(alignment: .center),
         message: attributedMessage,
         messageFont: OnboardingDialogsContants.messageFont,
        customActionView: AnyView(actionView))
}
 ```           
without wrapping it in a `VStack` ?

Reply:
Good question… I don’t remember! It was a while ago ahahah
I’ll try and remove it